content/summits/sparse/meeting1.md

```diff
@@ -32,7 +32,7 @@ birds-of-a-feather (BoF)-style talks, followed by more focused discussion.
    - Hameer Abbasi: Pydata-sparse current and future work
    - Jim Kitchen: needs/roles of sparse in GraphBLAS
    - Isaac Virshup: needs/roles of sparse in scverse
-   - Julien Jerphanion: needs/roles of sparse in `scikit-learn`
+   - [Julien Jerphanion: usage and needs for sparse data in `scikit-learn`](https://docs.google.com/presentation/d/1wsl03LXpmmhlKKkcnKUToCq0xaaLPyAmXP9vypjvsfg/edit?usp=sharing)
```

Comment:
I think it would be better to self host such things. It's hard to keep track of external links.

Reply:
I agree. Where would you see it self-hosted?

Reply:
We could have it in this repo or another one. This is more a question for @stefanv and @jarrodmillman 😃

Reply:
I will set up something for this soon.

Reply:
Sorry for this interjection, @jarrodmillman, but it https://github.com/scientific-python/scientific-python.org-blobs meant to store blobs of such presentations as well?